File: swex/add-in/opengl/OglTetrahedron/OglTetrahedron/AddIn.vb
'**********************
'Render Tetrahedron And Handle View Display Modes
'Copyright(C) 2019 www.codestack.net
'License: https://github.com/codestackdev/solidworks-api-examples/blob/master/LICENSE
'Product URL: https://www.codestack.net/solidworks-api/adornment/opengl/display-mode-tetrahedron/
'**********************

Imports CodeStack.SwEx.AddIn
Imports CodeStack.SwEx.AddIn.Attributes
Imports System.Runtime.InteropServices

<ComVisible(True), Guid("A28F5BB7-E468-48B6-9BBD-9E7A31FF8CC8")>
<AutoRegister("OpenGL Box Tetrahedron")>
Public Class AddIn
	Inherits SwAddInEx

	Public Overrides Function OnConnect() As Boolean
		CreateDocumentsHandler(Of OpenGlDocumentHandler)()
		Return True
	End Function
End Class

File: swex/add-in/opengl/OpenGlBoxGrid/OpenGlBoxGrid/AddIn.cs
//**********************
//Render Box Grid With Transparency
//Copyright(C) 2019 www.codestack.net
//License: https://github.com/codestackdev/solidworks-api-examples/blob/master/LICENSE
//Product URL: https://www.codestack.net/solidworks-api/adornment/opengl/box-grid-transparency/
//**********************

using CodeStack.SwEx.AddIn;
using CodeStack.SwEx.AddIn.Attributes;
using System;
using System.Runtime.InteropServices;

namespace CodeStack.OpenGlBoxGrid
{
    [ComVisible(true), Guid("FAB0F03B-785E-4113-B120-E18E7C73B9EB")]
    [AutoRegister("OpenGL Box Grid")]
    public class AddIn : SwAddInEx
    {
        public override bool OnConnect()
        {
            CreateDocumentsHandler<OpenGlDocumentHandler>();
            return true;
        }
    }
}
